src/components/tasks/card/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FunctionComponent } from 'react';
+import classNames from './card.scss'
+
+type pullRequest = {
+  title: string,
+  completionDate: string,
+  startedAt: string,
+  author: string,
+  profilePicture: string,
+  issueStatus: string
+}
+
+type Props = {
+  pullRequest: pullRequest
+}
+
+const Card: FunctionComponent<Props> = ({ pullRequest }) => {
+  const {
+    title,
+    completionDate,
+    startedAt,
+    author,
+    profilePicture,
+    issueStatus
+  } = pullRequest
+
+  return (
+    <div className={classNames.card}>
+      <span className={classNames.prTitle}>{title}</span>
+      <span className={classNames.statusElement}>
+        <span className={classNames.statusLable}>Estimated completion:</span><strong> {completionDate}</strong>
+      </span>
+      <span className={classNames.statusElement}>
+        <span className={classNames.statusLable}>Started:</span><strong> {startedAt}</strong>
+      </span>
+      <div className={classNames.cardFooter}>
+        <div className={classNames.profilePicture}>
+          <img src={profilePicture} alt="" />
+          <strong>{author}</strong>
+        </div>
+        <span className={classNames.statusElement}>
+          <span className={classNames.statusLable}>Status:</span><strong> {issueStatus}</strong>
+        </span>
+      </div>
```

Comment:
`<span className={classNames.statusElement}>
        <span className={classNames.statusLable}>Started:</span><strong> {startedAt}</strong>
      </span>`

I think we can re-use this .. its being repeated 3 times. Lets create a function on this same file and return this `span` tags from it